extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -197,9 +197,8 @@ export class ConsentUI {
         classList.add('amp-hidden');
       }
 
-      // Need to remove from fixed layer and add it back to update element's top
-      this.baseInstance_.getViewport().removeFromFixedLayer(this.parent_);
       toggle(dev().assertElement(this.ui_), false);
+      this.baseInstance_.getViewport().updateFixedLayer();
```

Comment:
toggling already updates the fixed layer.

Reply:
The element that's added to fixed layer is `this.parent_`, but I am toggling the `this.ui_` here. Thought it would not update the fixed layer? 
Does fixed layer update it self on mutation [hidden] change of every child?